rest/_examples/datasets.go

```diff
@@ -0,0 +1,117 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"gopkg.in/ns1/ns1-go.v2/rest/model/dataset"
+	"io"
+	"log"
+	"net/http"
+	"os"
+	"strings"
+	"time"
+
+	api "gopkg.in/ns1/ns1-go.v2/rest"
+)
+
+var client *api.Client
+
+// Helper that initializes rest api client from environment variable.
+func init() {
+	k := os.Getenv("NS1_APIKEY")
+	if k == "" {
+		fmt.Println("NS1_APIKEY environment variable is not set, giving up")
```

Comment:
Do you intend to panic here and missing the code for it or is `giving up` only for a logging purpose?

Reply:
It's just for logging proposes. This is what all files in the examples folder have